apps/AEPSampleApp/extensions/ConsentView.js

```diff
@@ -63,4 +62,30 @@ function setDefaultConsent(allowed: boolean) {
   var collectConsentStatus = allowed ? {"val": "y"} : {"val": "n"};
   var defaultConsents: {[keys: string]: any} = {"consent.default": {"consents" : {"collect" : collectConsentStatus}}};
   MobileCore.updateConfiguration(defaultConsents);
-}
\ No newline at end of file
+}
```

Comment:
revert the style change in Edge branch. Will do it in a Consent PR.

Reply:
should we revert this change?

Reply:
This is reverting the change for Consent View. Having all the styles back.  
I removed them from Consent PR.